dashboard/app/controllers/api/v1/sections_students_controller.rb

```diff
@@ -10,4 +12,25 @@ def index
       )
     end)
   end
+
+  # PATCH /sections/<section_id>/student/<id>/update
+  def update
+    student = @section.students.find_by_id(params[:id])
```

Comment:
Note, you can instead do this with CanCan by adding:

```ruby
load_resource :student, through: :section, only: :update
```
on line 3 above. Either way is fine now that it's just one action, but if we start adding more member actions it would be worth consolidating that query logic.